cmd/cluster/azure/create.go

```diff
@@ -371,7 +372,15 @@ func CreateInfraOptions(ctx context.Context, azureOpts *ValidatedCreateOptions,
 }
 
 // lookupRHCOSImage looks up a release image and extracts the RHCOS VHD image based on the nodepool arch
-func lookupRHCOSImage(ctx context.Context, arch string, image string, pullSecretFile string) (string, error) {
+func lookupRHCOSImage(ctx context.Context, arch, image, releaseStream, pullSecretFile string) (string, error) {
```

Comment:
It could be possible to cover the function with a unit test?

Reply:
This fix is blocking our azure e2e so I would prefer to add a unit test in a separate pr later on.

Reply:
This is done in the core create
https://github.com/openshift/hypershift/blob/cbf39ff40a564fc4a99dfd7b0c4e1b335538d39d/cmd/cluster/core/create.go#L210-L216

It seems that Azure needs this at the infra create stage, which is why we need to do it here?